### PR TITLE
#13520: Cleanup Backward pybind files

### LINKED ISSUE #13520: Cleanup Backward pybind files
PR : https://github.com/tenstorrent/tt-metal/pull/13522

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Work Done : 

- Fix Indentation for table format.
- Update backward Pybind files.

https://github.com/tenstorrent/tt-metal/actions/runs/11212285231 - passed

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp [hunk 1]
```diff
@@ -22,31 +22,24 @@ namespace binary_backward {
...
-
-        Supported dtypes, layouts, and ranks:
-
-
-        {3}
-
+        Note:
+            {3}
 
         Note : bfloat8_b/bfloat4_b is only supported on TILE_LAYOUT
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp:44]: Remove this Note

File: ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp [hunk 2]
```diff
@@ -111,11 +103,8 @@ void bind_binary_backward_int_default(py::module& module, const binary_backward_
...
 
-        Supported dtypes, layouts, and ranks:
-
-
-        {6}
-
+        Note:
+            {6}
 
         Note : bfloat8_b/bfloat4_b is only supported on TILE_LAYOUT
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/binary_backward/binary_backward_pybind.hpp:109]: Here too

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp [hunk 3]
```diff
@@ -748,13 +751,13 @@ void bind_unary_backward_shape(
 
 template <typename unary_backward_operation_t>
 void bind_unary_backward_optional(
-    py::module& module, const unary_backward_operation_t& operation, std::string_view description, std::string_view supported_dtype = "") {
+    py::module& module, const unary_backward_operation_t& operation, const std::string_view description, std::string_view supported_dtype = "") {
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp:804]: const std::string_view supported_dtype = ""

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp [hunk 4]
```diff
@@ -524,19 +528,18 @@ void bind_unary_backward_float_string_default(
...
-            grad_tensor (ttnn.Tensor): the input tensor.
+            grad_tensor (ttnn.Tensor): the input gradient tensor.
             input_tensor_a (ttnn.Tensor): the input tensor.
-            input_tensor_b (float): the input scalar.
-
+            {2} (float): {3}.
 
         Keyword args:
-            round_mode (round_mode, optional): Round mode for the operation. Defaults to `None`.
+            {4} (string, optional): {5} , Default value = {6}
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp:542]: Make this Defaults to {6}.

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp [hunk 5]
```diff
@@ -946,21 +1001,21 @@ void bind_unary_backward_gelu(
     const std::string& parameter_name_a,
     const std::string& parameter_a_doc,
     string parameter_a_value,
-    std::string_view description) {
+    const std::string_view description) {
     auto doc = fmt::format(
-        R"doc({0}(grad_tensor: ttnn.Tensor, input_tensor: ttnn.Tensor, {2}: string, *, memory_config: ttnn.MemoryConfig) -> std::vector<Tensor>
-
+        R"doc({0}(
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp:1006]: Remove `{0}(`

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp [hunk 6]
```diff
@@ -946,21 +1001,21 @@ void bind_unary_backward_gelu(
...
-            * :attr:`input_tensor`
+            grad_tensor (ttnn.Tensor): the input gradient tensor.
+            input_tensor_a (ttnn.Tensor): the input tensor.
 
         Keyword args:
-            * :attr:`{2}` (string): {3} , Default value = {4}
-            * :attr:`memory_config` (Optional[ttnn.MemoryConfig]): memory config for the output tensor
-            * :attr:`output_tensor` (Optional[ttnn.Tensor]): preallocated output tensor
-            * :attr:`queue_id` (Optional[uint8]): command queue id
+            {2}` (string): {3} , Default value = {4}.
```
Comment (mcw-anasuya) [inline ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp:1014]: Defaults to `{4}`.

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp (hunk 3 above)
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward_pybind.hpp]: I guess the code has been changes and this is from a different commit.

**Review by mcw-anasuya — APPROVED** [review]
LGTM

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by umadevimcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11226072912 - passed

**Pull request merged**